config/routes.rb

```diff
@@ -2416,6 +2416,16 @@
         save_post
     },
 
+    :volume_mapping => {
+      :get  => %w[
+        download_data
+        download_summary_pdf
+        show
+        show_list
+      ] +
+        compare_get
```

Comment:
this route is needed only for CI's that support compare feature, you can remove this route.

Reply:
@h-kataria 
What's a "CI"?